src/main/java/com/entropyteam/entropay/users/auth/models/User.java

```diff
@@ -0,0 +1,73 @@
+package com.entropyteam.entropay.users.auth.models;
+
+import java.util.Set;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import com.entropyteam.entropay.users.auth.common.BaseEntity;
+
+@Entity
+@Table(name = "[user]")
+public class User extends BaseEntity {
+
+    private String username;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String externalId;
```

Comment:
what is externalId?

Reply:
i thought i was going to need it, removing, good catch